third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -93,21 +93,21 @@ func TestAccContainerCluster_basic(t *testing.T) {
 				),
 			},
 			{
-				ResourceName:		 "google_container_cluster.primary",
-				ImportStateId: fmt.Sprintf("us-central1-a/%s", clusterName),
-				ImportState:		 true,
-				ImportStateVerify:	 true,
+				ResourceName:		   "google_container_cluster.primary",
```

Comment:
nit: not a big deal but i'm still seeing some spacing  weirdness here (it's a tabs/spaces thing)